### PR TITLE
WL-3268 : Change text and make LTI launch link into Oxford button

### PR DESCRIPTION
**Pull request by nicholaswilson100**

### CODE REVIEW

File: basiclti-portlet/src/java/org/sakaiproject/portlets/IMSBLTIPortlet.java [hunk 1]
```diff
@@ -116,10 +116,17 @@
 
 	public final static String CURRENT_HTTP_REQUEST = "org.sakaiproject.util.RequestFilter.http_request";
 
+	public static final String SITE_NAME = "ui.service";
+
+	public static final String SAKAI = "Sakai";
+
+	private org.sakaiproject.component.api.ServerConfigurationService serverConfigurationService;
```
Comment (buckett) [inline basiclti-portlet/src/java/org/sakaiproject/portlets/IMSBLTIPortlet.java]: Add an import?

**Comment by nicholaswilson100** [comment]
I don't think you can import two same-named classes in the same file using two import statements

File: basiclti-portlet/src/java/org/sakaiproject/portlets/IMSBLTIPortlet.java (hunk 1 above)
Comment (buckett) [inline basiclti-portlet/src/java/org/sakaiproject/portlets/IMSBLTIPortlet.java]: You shouldn't be using both the cover and the api in the same class, either change everything to use the api or use the cover for the new code. It's confusing to have them both used in the same class.

Comment (nicholaswilson100) [inline basiclti-portlet/src/java/org/sakaiproject/portlets/IMSBLTIPortlet.java]: Ok.  Should we in general be using cover always or api always? i.e., which is better to consistently use?

Comment (buckett) [inline basiclti-portlet/src/java/org/sakaiproject/portlets/IMSBLTIPortlet.java]: The covers are all deprecated, but if there are lots of references to a class it's sometimes easier just to carry on using the cover for a small fix.

**Comment by nicholaswilson100** [comment]
Cool thanks

**Pull request merged**